src/frameworks/drupal8/drush.md

```diff
@@ -20,54 +12,72 @@ Repository](https://github.com/drush-ops/drush#description)
 >
 > Platform's CLI requires **Drush 6 or greater**.
 
-## Install Drush locally
+## Installing Drush on Platform.sh
 
-Install drush with Composer:
+### Install Drush in Platform.sh-provided Drupal examples
 
-```bash
-$ composer global require drush/drush
-```
+Platform.sh's Drupal templates have Drush installed automatically via Composer and should not require you to add it.
 
-At the end of the installation, you should be able to run:
+### Install Drush in custom projects using _Composer_
 
+Run this command in the project's repository root folder:
 ```bash
-$ drush
+$ composer require drush/drush
 ```
+Then, commit and push.
 
-And see a list of available commands.
+Drush will then be available at `vendor/bin/drush`, in the exact same version on your local system and on Platform.sh.
 
-> -   [Drush repository on Github](https://github.com/drush-ops/drush)
+### Install Drush in custom projects using _Build Dependencies_
```

Comment:
Do we even want to document this option?  For Drupal 8 users should pretty much always use the composer approach.

Reply:
What if they aren't using Composer?